Source: https://www.mail-archive.com/debian-bugs-dist@lists.debian.org/msg1964644.html

Description: Add declarations for functions to fix implicit function
declaration errors.
Author: Peter Michael Green <plugw...@debian.org>
---
--- gtk2-engines-murrine-0.98.2.orig/src/murrine_rc_style.h
+++ gtk2-engines-murrine-0.98.2/src/murrine_rc_style.h
@@ -155,4 +155,6 @@ struct _MurrineRcStyleClass
 
 GType murrine_rc_style_get_type       (void);
 
+void murrine_rc_style_register_types (GTypeModule *module);
+
 #endif /* MURRINE_RC_STYLE_H */
--- gtk2-engines-murrine-0.98.2.orig/src/murrine_style.h
+++ gtk2-engines-murrine-0.98.2/src/murrine_style.h
@@ -102,5 +102,6 @@ struct _MurrineStyleClass
 };
 
 GType murrine_style_get_type (void);
+void murrine_style_register_types (GTypeModule *module);
 
 #endif /* MURRINE_STYLE_H */
--- gtk2-engines-murrine-0.98.2.orig/src/support.h
+++ gtk2-engines-murrine-0.98.2/src/support.h
@@ -149,4 +149,7 @@ G_GNUC_INTERNAL void murrine_get_noteboo
                                                         gboolean  *start,
                                                         gboolean  *end);
 
+gboolean murrine_object_is_a (const GObject * object, const gchar * type_name);
+gboolean murrine_widget_is_ltr (GtkWidget *widget);
+
 #endif /* SUPPORT_H */
